Release/src/websockets/client/ws_client_wspp.cpp

```diff
@@ -724,8 +724,8 @@ class wspp_callback_client : public websocket_client_callback_impl,
             case websocket_message_type::binary_message:
                 client.send(this_client->m_con, sp_allocated.get(), length, websocketpp::frame::opcode::binary, ec);
                 break;
-            case websocket_message_type::ping: client.ping(this_client->m_con, "", ec); break;
-            case websocket_message_type::pong: client.pong(this_client->m_con, "", ec); break;
+            case websocket_message_type::ping: client.ping(this_client->m_con, *(std::string*)sp_allocated.get(), ec); break;
```

Comment:
Is that cast OK? Isn't `sp_allocated` a pointer to raw bytes, not a `std::string` object?
Since `ping` and `pong` only accept `std::string` we probably need to construct one here something like: `std::string(sp_allocated.get(), length)`?

Reply:
I’ll change that